src/TaskGroup.h

```diff
@@ -11,11 +11,13 @@
 
 #include "HasTaskSet.h"
 #include "TaskishUid.h"
+#include "ThreadDb.h"
```

Comment:
Done.  Sorry about that, it was mentioned in the previous review and I didn't deal with it.